n3fit/src/n3fit/layers/msr_normalization.py

```diff
@@ -1,11 +1,18 @@
 from n3fit.backends import MetaLayer
 from n3fit.backends import operations as op
 
-GLUON_IDX = [[2]]
-V_IDX = [[3], [7], [8]]
-V3_IDX = [[4]]
-V8_IDX = [[5]]
-V15_IDX = [[6]]
+
+IDX = {
+    'photon': 0,
+    'sigma': 1,
+    'g': 2,
+    'v': 3,
+    'v3': 4,
+    'v8': 5,
+    'v15': 6,
+    'v35': 7,
+    'v24': 8,
+}
```

Comment:
It doesn't affect the result, but the order of `v35` and `v24` is reversed.

Since we rely on hardcoding the basis of the fktable anyway, it may be best to have a `dict` like this one stored in a single location which all modules that need it can then import (which at the moment I think are just `x_operations.py`, the current module and `writer.py`).

Reply:
I agree, do you want me to do that in this commit? If so I'm not sure where to put it.

Reply:
That's also what I'm not too sure about, perhaps we should add an `n3fit/utils.py`? @scarlehoff do you have any ideas?

Reply:
The information should be here https://github.com/NNPDF/nnpdf/blob/9cf5f9ecc83ec59fff804f859b531198139021fe/validphys2/src/validphys/pdfbases.py#L695

incidentally, the name of that function, `fitbasis_to_NN31IC`, is no longer correct.

Some things that we said (a very long time ago) someone should do are:
1. Create a function that generate the PDF output -> fktable rotation and use that on the fit (half of the work @APJansen  already did with  https://github.com/NNPDF/nnpdf/pull/1772)
2. Define in that module what the "final fitbasis" actually is so that both `fitbasis_to_NN31IC` and any other function can use that information.

Maybe `fitbasis_to_fitoutput` would be more correct. Or even skip that step and have just `fitbasis_to_fkbasis` since the `fitoutput->fkbasis` step is known and constant. That would also require (small) changes to the QED fit or the integration.

Reply:
This is probably better left to another PR?